### PR TITLE
Color Management: Documentation

### PR DESCRIPTION
**Pull request by donmccurdy**
Documentation accompanying #23392. Explains concepts required for three.js users to understand color management, including: 

- What is a color space?
    - ... and differences between sRGB and Linear-sRGB
- Roles of color spaces
    - ... and how to know which to use in different contexts
- Working with THREE.Color instances

Live Preview: [three.js / docs / getting started / color management](https://raw.githack.com/donmccurdy/three.js/feat-color-management-v4-pt2/docs/#manual/en/introduction/Color-management)

/cc @gkjohnson @WestLangley @Mugen87 

***

_Aside:_ I'm very grateful to @WestLangley for explaining foundational color science concepts to me, and to @sobotka for correcting my (many) misconceptions in this process. Any mistakes in this PR are of course mine, and I hope the new documentation can pay their help forward to the three.js community somewhat. ☺️

### CODE REVIEW

**Comment by nyan-left** [comment]
Nice work @donmccurdy !

I'm a little confused about this part:
```
[Scene.fog]() and [Scene.background]() are exceptions to the rule. Both are unaffected by 

[WebGLRenderer.outputEncoding]() and so must store RGB components in the renderer's output color space.
```

Is there any particular reason why this is the case?

**Comment by WestLangley** [comment]
@nyan-left See https://github.com/mrdoob/three.js/pull/11076#issuecomment-293567895.

**Comment by nyan-left** [comment]
Thanks @WestLangley

**Comment by WestLangley** [comment]
oops. reconstructing the essence of a side discussion between @nyan-left and @WestLangley...

@WestLangley said:

>[This comment](https://github.com/mrdoob/three.js/pull/11076#issuecomment-293567895) provides the reason why fog and clear-color are currently a special case.

File: docs/manual/en/introduction/Color-management.html [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+	</ul>
+
+	<blockquote>
+		<p>
+			⚠️ <i><b>WARNING:</b> Render targets may use either sRGB or Linear-sRGB. sRGB makes
+			better use of limited precision — 8 bits suffice for LDR output to sRGB, whereas ≥12 bits
+			(half float textures) are often required for LDR output to Linear-sRGB. However, the
+			next pipeline stage using the render target may require input in the working color
+			space, Linear-sRGB, incurring modest overhead to perform an additional sRGB → Linear-sRGB
+			conversion.</i>
```
Comment (donmccurdy) [inline docs/manual/en/introduction/Color-management.html:252]: TODO: Use of the term "LDR" is ambiguous here, closed domain [0,1] would be clearer.

**Comment by donmccurdy** [comment]
This PR is ready for review — given the changes coming in #23392, publishing the these docs with r139 might be ideal.

@WestLangley — no pressure, but I'm happy to make any changes you'd recommend here!

**Comment by mrdoob** [comment]
Thanks!

**Comment by WestLangley** [comment]
@donmccurdy Better late than never... I just want to say thank you for all your work on this. :-)

**Pull request merged**